packages/studio-ui/src/web/util/InjectSegment.ts

```diff
@@ -48,17 +48,21 @@ export default (user: UserReducer): void => {
   })
 }
 
-function trackEvent(eventName: string, payload?: any, options?: any, callback?: (...params: any[]) => any) {
+function trackEvent(eventName: string, payload?: any, options?: any, callback?: (...params: any[]) => any): void {
   if (analytics) {
     // analytics only defined if window.SEND_USAGE_STATS is true
-    return analytics.track(eventName, payload, options, callback)
+    analytics.track(eventName, payload, options, callback).catch(e => {
+      console.error('Analytics error - event', e)
```

Comment:
Do we really want to log those errors? I don't think it has much value to let the user know that sending analytics failed. What do you think?